app/src/main/java/com/example/wrk/fragments/ProfileFragment.java

```diff
@@ -177,6 +197,30 @@ public void onClick(View v) {
         queryWorkouts();
     }
 
+    private double calcPercentage() throws ParseException {
+        ParseUser currentUser = ParseUser.getCurrentUser();
+        List<ParseUser> followingList = getFollowing();
+        List<ParseUser> friendList = new ArrayList<>();
+        // to ensure that signed in user is not included in the calculations (they follow themselves by default)
```

Comment:
in that case, do you want to just substract instead of checking everytime? or is this list not guaranteed to not include duplicates?

Reply:
> in that case, do you want to just subtract 1 instead of checking every time? or is this list not guaranteed to not include duplicates?

Well, it may not be guaranteed that the current user's position in the list is first/last so I figured that I'd need to check every time. There shouldn't be duplicates within the list though.